hbase-client/src/main/java/org/apache/hadoop/hbase/MetaTableAccessor.java

```diff
@@ -1050,6 +1050,9 @@ public static RegionInfo getRegionInfo(Result data) {
    */
   @Nullable
   private static RegionInfo getRegionInfo(final Result r, byte [] qualifier) {
+    if (r == null){
```

Comment:
What does this change add? We'll get a NPE when we try to use it on line #1056 anyways, if r is null.